### DISCUSSION TITLE
How to check issue log when "Copilot is encountering temporary issues"

### DISCUSSION THREAD

**Discussion by yifanff** [community/community#10715] [Copilot Conversations] [answered]
The Copliot is not work as the image below. When the Copilot is encoutering temporary issues from VScode as below, where can i find the issue log and trouble shoot?

![image](https://user-images.githubusercontent.com/5958693/151649583-bf8c8687-e6cf-4900-ab23-332b901aefd7.png)

**Comment by AcmeGamers** [comment]
Okay so I actually encountered this issue, the solution is very straight forward and it should be fixed by now if you open VS Code. In case it doesn't, here's the fix:

## Possible Solutions
* Simply reload VS Code by closing it using **Ctrl + Shift + W**, and then opening VS Code again in a new window.

Or (the solution that worked for me)

* You can try closing unnecessary applications, and then reopen VS Code.

## Conclusion
So a moment ago, I actually put a lot of load over my laptop, I closed unwanted apps, including my VS code using **Ctrl + Shift + W**, and after I ran it, the problem was gone as if it never existed. I hope this helps you out.

**Comment by davecheney** [comment, marked as answer]
View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by yifanff** [reply]
Thanks.
